src/otx/algo/detection/heads/anchor_head.py

```diff
@@ -75,26 +79,23 @@ def __init__(
             raise ValueError(msg)
         self.reg_decoded_bbox = reg_decoded_bbox
 
-        self.bbox_coder = TASK_UTILS.build(bbox_coder)
-        self.loss_cls = MODELS.build(loss_cls)
-        self.loss_bbox = MODELS.build(loss_bbox)
+        self.bbox_coder = DeltaXYWHBBoxCoder(**bbox_coder)
+        self.loss_cls = CrossSigmoidFocalLoss(**loss_cls)
+        self.loss_bbox = GIoULoss(**loss_bbox)
```

Comment:
What if we avoid hard-coding these modules? AnchorHead is shared with MaskRCNN, but not all modules are identical. Instead of hard-coding, could we consider branching? For example:
```python
if loss_bbox["type"] == GIoULoss:
    self.loss_bbox = GIoULoss(**loss_bbox)
elif loss_bbox["type"] == L1Loss:
    self.loss_bbox = L1Loss(**loss_bbox)
else:
    raise NotImplementedError
```
It's more flexible and allows for other variations in modules. Hard-coding is risky as not every anchor head is used globally by different mmdet models.

Reply:
You're right. I'll change hard code build to use build_*** function during refactoring. Refactoring will be done next week.
So this will be changed to
`self.loss_bbox = build_loss_bbox(**loss_bbox)`